WordPress/Classes/ViewRelated/NUX/LoginViewController.swift

```diff
@@ -23,6 +23,10 @@ class LoginViewController: NUXViewController, SigninWPComSyncHandler, LoginFacad
         }
     }
 
+    @objc
```

Comment:
Is this `@objc` necessary?

Reply:
No sir, it is not! Nice catch, thanks!. Pushing an update in a minute.